.github/scripts/openscad.ps1

```diff
@@ -0,0 +1,14 @@
+# Prepare the registry
+reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem" /v LongPathsEnabled /t REG_DWORD /d 1 /f
+
+# Install OpenSCAD
+curl -o openscad.zip https://files.openscad.org/OpenSCAD-2021.01-x86-64.zip
+curl -o openscad.zip.sha256 https://files.openscad.org/OpenSCAD-2021.01-x86-64.zip.sha256
+# certutil -hashfile openscad.zip SHA256 > computed.sha256
```

Comment:
This part was tested in `pwsh` on Linux and it does not have [`certutil`](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/certutil).